metadata.json

```diff
@@ -60,10 +60,6 @@
     }
   ],
   "requirements": [
-    {
-      "name": "pe",
-      "version_requirement": ">= 3.2.0"
-    },
     {
       "name": "puppet",
       "version_requirement": ">= 3.2.0"
```

Comment:
Is it realy puppet `3.2.0` ?

Reply:
Right now, probably.  I need to do a proper pass on the metadata.json file to update for VP.  The commit that modified it here was just to get the tests passing.